scripts/prepend-go-build-to-tests.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+# prepend-go-build-to-tests.sh prepends '//go:build <BUILD_TAG>\n\n' to '*_test.go' files within
+# the directory it is run as well as subdirectories as long as those files don't already starts
+# with '//go:build' or '// Code generated by'
+
+usage() {
+  echo "usage: $0 <BUILD_TAG>"
+  exit 2
+}
+
+BUILD_TAG=$1
+if [[ -z $BUILD_TAG ]]; then
+  usage
+fi
+
+for file in $(git ls-files -- '*_test.go'); do
+	if [[ $(sed -n '/^\/\/go:build/p;q' "${file}") ]]; then
+		#echo "${file} starts with //go:build"
+		continue
+	elif [[ $(sed -n '/^\/\/ Code generated by/p;q' "${file}") ]]; then
+		#echo "${file} is generated code that should not be touched"
```

Comment:
Leftover debug output?

Reply:
Yes, they were certainly useful when writing and testing the script but I'll remove them both now.